Src/ILGPU.Algorithms/Runtime/Cuda/CuFFTWConstants.cs

```diff
@@ -0,0 +1,34 @@
+﻿// ---------------------------------------------------------------------------------------
+//                                   ILGPU Algorithms
+//                           Copyright (c) 2021 ILGPU Project
+//                                    www.ilgpu.net
+//
+// File: CuFFTWConstants.cs
+//
+// This file is part of ILGPU and is distributed under the University of Illinois Open
+// Source License. See LICENSE.txt for details.
+// ---------------------------------------------------------------------------------------
+
+#pragma warning disable CA1707 // Identifiers should not contain underscores
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+
+namespace ILGPU.Runtime.Cuda
+{
+    public static class CuFFTWConstants
```

Comment:
Question: these constants look dangerous... Should they be exposed?

Reply:
Unfortunately, these are needed in the low-level API for CuFFTW.